src/tox/config/parallel.py

```diff
@@ -60,7 +66,6 @@ def add_parallel_flags(parser):
         help="connect to stdout while running environments",
     )
 
-
```

Comment:
This change is unrelated and wrong. As per PEP 8, it must be two line separator between higher-level functions.